src/main/java/ch/njol/skript/log/RedirectingLogHandler.java

```diff
@@ -38,6 +39,8 @@
 	
 	private int numErrors = 0;
 	
+	private long timeTaken = System.currentTimeMillis();
```

Comment:
You should create a new log handler to record the time instead of adding it to a random one